snuba/manual_jobs/job_status.py

```diff
@@ -0,0 +1,9 @@
+from enum import StrEnum
```

Comment:
job_status.py was created to avoid circular dependency

Reply:
where were you having a circular import?

Reply:
`redis.py` imports `__init__.py/JobStatus`, `__init__.py` imports `redis.py?_set_job_type`